common/v2/features/SendAssets/components/ConfirmTransaction.tsx

```diff
@@ -97,15 +104,24 @@ export default function ConfirmTransaction({
           <img src={sendIcon} alt="Send" /> Send Amount:
         </div>
         <div className="ConfirmTransaction-row-column">
-          <Amount assetValue={`${amount} ${asset.ticker}`} fiatValue="$1" />
+          <Amount
+            assetValue={`${amount} ${asset.ticker}`}
+            fiatValue={`$${calculateValue(getRate(asset.ticker as TTicker), amount).toFixed(2)}`}
+          />
         </div>
       </div>
       <div className="ConfirmTransaction-row">
         <div className="ConfirmTransaction-row-column">
           <img src={feeIcon} alt="Fee" /> Max. Transaction Fee:
         </div>
         <div className="ConfirmTransaction-row-column">
-          <Amount assetValue={`${maxTransactionFeeBase} ${baseAsset.ticker}`} fiatValue="$1" />
+          <Amount
+            assetValue={`${maxTransactionFeeBase} ${baseAsset.ticker}`}
+            fiatValue={`$${calculateValue(
+              getRate(baseAsset.ticker as TTicker),
+              maxTransactionFeeBase
+            ).toFixed(2)}`}
```

Comment:
Since we are repeating this multiple times, how about:

```
const convertToDecimal = rate => balance => convertToFiat(balance, rate).toString(2);
const convertFromAsset = balance => convert(getRate(asset.ticker));
const convertFromBase = balance => convert(getRate(baseAsset.ticker));
// or any name that is more expressive
...
  fiatValue={`convertFromBase(maxTransactionFeeBase)`}

```